bug/snapshot.go

```diff
@@ -62,3 +64,27 @@ func (snap *Snapshot) SearchTimelineItem(hash git.Hash) (TimelineItem, error) {
 
 	return nil, fmt.Errorf("timeline item not found")
 }
+
+// append the operation author to the actors list
+func (snap *Snapshot) addActor(actor identity.Interface) {
+	for _, a := range snap.Actors {
```

Comment:
Maybe should i just store actors and participants in a `map[string]identity.Interface` to avoid looping each time `addActor` ?

Reply:
As the Snapshot API would stay the same, this is really a question of performance. Is it better to use more CPU, or more memory ? I'd say it's better this way, as it avoid any memory allocation, and no GC as well (so less CPU !).